docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -11,14 +11,16 @@
     "(realtime-monitor-drift-tutor)=\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    !pip install "evidently=={SUPPORTED_EVIDENTLY_VERSION}"
</pre><p>we may add this line as an exception that if the has <code>evidently</code> it will skip the pip install  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>